crates/system/src/lib.rs

```diff
@@ -18,34 +19,25 @@ pub type Error = String;
 pub type EventProcessingResult<T> = Result<T, Error>;
 
 pub struct EventProcessor {
-    pub events_queue: Arc<dyn EventsQueue>,
-    quests_channel: Arc<Mutex<dyn QuestsChannel>>,
+    pub events_queue: Arc<dyn MessagesQueue<Event>>,
+    quests_channel: Arc<Mutex<dyn ChannelPublisher<UserUpdate>>>,
```

Comment:
In the server crate we are using `RedisMessagesQueue`, is there a reason to do it dynamically here?

Reply:
Nope, I just replicated it as it was for the other components